### ISSUE TITLE
u256 on top of u128

### ISSUE THREAD

**Issue by k06a** [MaxGraey/bignum.wasm#71]
Why not to build `u256` on top of `u128`? Implementation will be the same as `u128` on top of `u64`. Implementation can even be templated by subtype.

**Comment by MaxGraey** [comment]
I wasn't done for performance reasons. as-bignum exclusively focused on perf. Building over `u128` will cause to significant  overhead

**Comment by k06a** [comment]
@MaxGraey agree, but I would prefer to have working implementation first and optimized shipped with update.
